tests/Shared/GetTestProjects.proj

```diff
@@ -0,0 +1,44 @@
+<Project DefaultTargets="GenerateListOfTestsForGithubActions">
+  <!--
+    This is meant to be invoked directly to get the list of test projects to run for PR validation.
+    It emits "shortnames" for the project which is essentially the project name with `Aspire.` prefix
+    `.Tests` removed.
+
+    Input: $(TestsListOutputPath) -->
+
+  <Target Name="GenerateListOfTestsForGithubActions">
+    <Error Condition="'$(TestsListOutputPath)' == ''"
+           Text="%24(TestsListOutputPath) is unset" />
+
+    <PropertyGroup>
+      <RepoRoot>$(MSBuildThisFileDirectory)..\..\</RepoRoot>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <_TestProjectsToExclude Include="$(RepoRoot)tests\Shared\**\*Tests.csproj" />
+      <_TestProjectsToExclude Include="$(RepoRoot)tests\testproject\**\*Tests.csproj" />
+      <_TestProjectsToExclude Include="$(RepoRoot)tests\TestingAppHost1\**\*Tests.csproj" />
+
+      <_TestProjectsToExclude Include="$(RepoRoot)tests\Aspire.Workload.Tests\**\*Tests.csproj" />
+      <_TestProjectsToExclude Include="$(RepoRoot)tests\Aspire.EndToEnd.Tests\**\*Tests.csproj" />
+
+      <_TestProjects Include="$(RepoRoot)tests\**\*Tests.csproj"
+                     Exclude="@(_TestProjectsToExclude)" />
+    </ItemGroup>
+
+    <MSBuild Projects="@(_TestProjects)" Targets="GetRunTestsOnGithubActions">
+      <Output TaskParameter="TargetOutputs" ItemName="ProjectsToTest" />
+    </MSBuild>
```

Comment:
Doesn't that target return `true` or empty? What does it mean to have a bunch of `true` items?

Reply:
ugh.. yes, you are correct. Missed this in the rework. thank you!